README.md

```diff
@@ -2,39 +2,59 @@
 
 Here you will find all the [specification types](https://github.com/serverlessworkflow/specification/blob/master/schema/workflow.json) defined by our Json Schemas, in Go.
 
-Some types defined by the specification can be generic objects (such as [`Extensions`](https://github.com/serverlessworkflow/specification/blob/master/extending/README.md)) 
-or share a minimum interface, like [`States`](https://github.com/serverlessworkflow/specification/tree/master/README.md#State-Definition). 
+The final goal of the Serverless Workflow Specification is to offer SDKs that
+have similar features, so users would have the same experience no matter the chosen language.
+Go SDK is a _work in progress_, but should catch up with [Java SDK](https://github.com/serverlessworkflow/sdk-java) soon. 
 
-In cases like these, we've decided to represent their types as the Kubernetes type [`RawExtension`](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#rawextension),
-that can be found on `kuberbetes/serverlessworkflow` package. 
+Current status of features implemented in the SDK is listed below.
 
-This way the Serverless Workflow types can be Kubernetes friendly, to make it easy for one to
-use the package when developing Kubernetes applications.
+| **Feature**                                 | Java SDK           | Go SDK             |
```

Comment:
Again I would not do a comparison if possible. I think it would be nice tho if each sdk had a roadmap document. we can work on that to define one for both sdks if you want.